website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -2441,5 +2466,8 @@ var Range = ace.require('ace/range').Range;
         chunk.skipLines(2, 1, true);
     }
 
-
 })();
+$(function(){
+  $("wmd-help-button").attr("data-toggle","modal");

```

Comment:
Aren't you already setting these attributes here: https://github.com/CenterForOpenScience/osf.io/blob/9afaa0786610ef91601beab5e4bc558955f6309c/website/addons/wiki/static/pagedown-ace/Markdown.Editor.js#L1660-L1661 ?